frontend/public/style/_overrides.scss

```diff
@@ -213,53 +213,114 @@ tags-input .autocomplete .suggestion-item em {
 
 // PatternFly 4 nav overrides
 
-$pf-l-page__active-sidebar--BackgroundColor: rgba(255,255,255,.07);
+.pf-l-page__sidebar {
+  --pf-l-page__sidebar--BackgroundColor: #{$color-pf-black-900};
+  --pf-l-page__sidebar--BoxShadow: none;
+  --pf-l-page__sidebar--PaddingBottom: 0;
+  --pf-l-page__sidebar--PaddingTop: 0;
+  .pf-c-nav {
+    // List link
+    --pf-c-nav__list-link--active--after--BackgroundColor: var(--pf-c-nav__list-link--hover--after--BackgroundColor);
+    --pf-c-nav__list-link--active--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
+    --pf-c-nav__list-link--active--Color: var(--pf-c-nav__list-link--hover--Color);
+    --pf-c-nav__list-link--after--Height: 100%;
+    --pf-c-nav__list-link--after--Left: 0;
+    --pf-c-nav__list-link--after--md--Left: var(--pf-c-nav__list-link--after--Left);
+    --pf-c-nav__list-link--after--Width: 3px;
+    --pf-c-nav__list-link--Color: #{$color-pf-black-300};
+    --pf-c-nav__list-link--focus--after--BackgroundColor: var(--pf-c-nav__list-link--hover--after--BackgroundColor);
+    --pf-c-nav__list-link--focus--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
+    --pf-c-nav__list-link--focus--Color: var(--pf-c-nav__list-link--hover--Color);
+    --pf-c-nav__list-link--hover--after--BackgroundColor: transparent;
+    --pf-c-nav__list-link--hover--BackgroundColor: #{$color-pf-black-700};
+    --pf-c-nav__list-link--hover--Color: #{$color-pf-white};
+    --pf-c-nav__list-link--m-current--after--BackgroundColor: #{$color-pf-blue-300};
+    --pf-c-nav__list-link--m-current--Color: var(--pf-c-nav__list-link--hover--Color);
+    --pf-c-nav__list-link--md--PaddingLeft: 32px;
```

Comment:
This needs a media query around it as line 241 is always winning.  

<img width="335" alt="screen shot 2018-12-12 at 5 36 53 pm" src="https://user-images.githubusercontent.com/895728/49903386-08d99880-fe35-11e8-8fc4-d1150a3ce2d9.png">

Reply:
Hmm. I'm not seeing this. The media query that comes from PF4 is applying L239 on my end.

<img width="665" alt="screen shot 2018-12-12 at 4 46 35 pm" src="https://user-images.githubusercontent.com/35148959/49903726-c5c7f700-fe2d-11e8-8b82-72ed8deed0f6.png">
<img width="1149" alt="screen shot 2018-12-12 at 4 49 17 pm" src="https://user-images.githubusercontent.com/35148959/49903775-e2642f00-fe2d-11e8-9102-3fbb5448887b.png">
<img width="468" alt="screen shot 2018-12-12 at 4 49 08 pm" src="https://user-images.githubusercontent.com/35148959/49903776-e2642f00-fe2d-11e8-8fb3-839fd5a51dcc.png">

Reply:
Or rather 241 needs to be removed it seems?

Reply:
Never mind.  Build or cache error.